src/common/transformations/src/transformations/common_optimizations/clamp_fusion.cpp

```diff
@@ -48,6 +48,8 @@ ov::pass::ClampFusion::ClampFusion() {
 
         double min_value = min_const->cast_vector<double>()[0];
         double max_value = max_const->cast_vector<double>()[0];
+        if (min_value > max_value)
+            return false;
```

Comment:
do we expect 0 as a result of Min, Max ops execution in this case?
what is the result of Clamp operation execution in this case?

Reply:
Why 0? It will be dependent on the order of operations. If max is first the result will be `min_value` in the opposite case it will be `max_value`.

Reply:
If min_value > max_value Clamp will fail on validation, but unfused ops will return correct results